services/compute/virtualmachine/virtualmachine.go

```diff
@@ -414,6 +414,7 @@ func (c *client) getVirtualMachine(vm *wssdcloudcompute.VirtualMachine, group st
 			VmType:                  vmtype,
 			DisableHighAvailability: &vm.DisableHighAvailability,
 			Host:                    c.getVirtualMachineHostDescription(vm),
+			NodeName:                &vm.NodeName,
```

Comment:
This is hidden for a reason. Why do we need this?
We cannot expose physical nodes to clients that creates vms